soccer/src/soccer/strategy/agent/position/defense.cpp

```diff
@@ -36,15 +36,15 @@ Defense::State Defense::update_state() {
     }
 
     switch (current_state_) {
-        case IDLING:
-            break;
-        case JOINING_WALL:
+        case IDLING:{
```

Comment:
this seems like it breaks code style (no space before '{' and no newline before '}'). did you run a formatter? it's also possible our formatter doesn't catch this case (blocks in a case statement is a bit nonstandard but I do love them)